formatTest/typeCheckedTests/expected_output/arityConversion.re

```diff
@@ -7,7 +7,8 @@ type bcd =
 let a = TupleConstructor (1, 2);
 
 let b =
-  MultiArgumentsConstructor 1 2 [@implicit_arity];
```

Comment:
This seemed to have nothing to do with comment interleaving. Do you know what happened? (Not a blocker)